src/api/entityStatus.ts

```diff
@@ -0,0 +1,12 @@
+import { EntityStatusResponse } from 'deps/control-plane/types';
+import { client } from 'services/client';
+import { getEntityStatusSettings } from 'utils/env-utils';
+
+const { entityStatusBaseEndpoint } = getEntityStatusSettings();
+
+// Local API documentation can be found here: http://localhost:8675/api/v1/docs
+export const getEntityStatus = async (
+    accessToken: string,
+    catalogName: string
+): Promise<EntityStatusResponse[]> =>
+    client(`${entityStatusBaseEndpoint}?name=${catalogName}`, {}, accessToken);
```

Comment:
I think probably best to do it here - but we need to make sure `catalogName` is safe / escaped. Normally we have used `escapeReservedCharacters` but that is really for PostgREST so not 100% sure what we wanna do here.